webhook.go

```diff
@@ -506,15 +506,16 @@ func (whsvr *WebhookServer) mutateInitialization(pod corev1.Pod, req *v1beta1.Ad
 
 	// Get list of images
 	images := ""
-	for _, cspec := range pod.Spec.InitContainers {
+	// Containers images should go first (since they are alphabetically before initContainers)
```

Comment:
I'm wondering if ordering matters here... not sure if it may lead to some problems in the future. Wonder if can just put images in a hash map and spit them out lexicographically... Really depends on the usecase of what the user wants to get out of images. I am fine with this for now... But I have the opposite intuition that initContainers go first, but I think it doesn't matter so i'm fine with it being arbitrary as long as it is consistent and definable.

Reply:
I created a new issue to reference it, "Consistent sorting of pod images" #67